corehq/apps/hqadmin/views.py

```diff
@@ -1134,3 +1137,68 @@ def get(self, request, *args, **kwargs):
             return JsonResponse(data)
         else:
             return HttpResponse('Only web users can access this endpoint', status=400)
+
+
+@method_decorator(csrf_exempt, name='dispatch')
+@method_decorator(validate_request_hmac('FORMPLAYER_INTERNAL_AUTH_KEY', ignore_if_debug=True), name='dispatch')
+class SessionDetailsView(View):
```

Comment:
@snopoke just discovered looking at https://rpm.newrelic.com/accounts/366142/applications/33065068/transactions#id=5b225765625472616e73616374696f6e2f537072696e67436f6e74726f6c6c65722f6e617669676174655f6d656e752028504f535429222c22225d and https://rpm.newrelic.com/accounts/366142/applications/5058236/transactions#id=5b225765625472616e73616374696f6e2f46756e6374696f6e2f636f726568712e617070732e687161646d696e2e76696577733a53657373696f6e44657461696c73566965772e706f7374222c22225d that this view is calling `couchdbkit.client:Database.save_doc` twice per call. I stuck a pdb statement in save_doc locally to see when it happens, and I traced it back to auditcare and the presence of `'corehq.apps.hqadmin'` in `AUDIT_MODULES`.

This seems to add a surprising amount to each request time, 80-200ms. If the auditing is helpful, that wouldn't be _that_ high a price to pay, but I'm guessing it was just unintentional and no one's interested in logging every formplayer authentication request. If we can get rid of it, a sizable majority (~90%) of formplayer requests appear to be under a second, making even 100ms off a noticeable improvement.

If you're on board with moving this but not enthused about any place in particular to keep this, I'd suggest the non-admin bucket "app", `hqwebapp`.